docs/UserGuide.md

```diff
@@ -304,19 +319,19 @@ _Details coming soon ..._
+| **Find person(s)**                    | `findcustomer`, `findc`     | `findcustomer KEYWORD [MORE_KEYWORDS]`<br> e.g. `findcustomer James Jake`                                                                                                           |
+| **List all persons**                  | `listcustomer`              |                                                                                                                                                                                     |
+| **Add an order**                      | `addorder`, `addo`          | `addorder INDEX i/ITEM d/DATE [q/QUANTITY]`<br> e.g. `addorder 2 i/Lamp d/20-11-2024 q/3`                                                                                           |
+| **List all orders**                   | `listorder`, `listo`        | `listorder INDEX`<br> e.g. `listorder 3`                                                                                                                                            |
+| **Delete an order**                   | `deleteorder`, `deleteo`    | `deleteorder ORDER_INDEX`<br> e.g. `deleteorder 2`                                                                                                                                  |
+| **Edit an order**                     | `editorder`, `edito`        | `editorder ORDER_INDEX [i/ITEM] [d/DATE] [q/QUANTITY]` <br> e.g. `editorder 1 i/Light bulb d/21-11-2025`                                                                            |
+| **Mark an order as completed**        | `markorder`, `marko`        | `markorder ORDER_INDEX`<br> e.g. `markorder 2`                                                                                                                                      |                                                                                                                                                                                                                                           
+| **Revert an order to pending status** | `unmarkorder`, `unmarko`    | `unmarkorder ORDER_INDEX`<br> e.g., `unmarkorder 2`                                                                                                                                 |
+| **Filter orders by status**           | `filterorder`, `filtero`     | `filterorder ORDER_STATUS`<br> e.g. `filterorder Completed`                                                                                                                         |
+| **Help**                              | `help`                      |                                                                                                                                                                                     |
```

Comment:
Not sure if necessary, but can include `exit` as well?

Reply:
added